### PR TITLE
Lab Sina Pedro

### PR DESCRIPTION
**Pull request by PJPalma**
Done Iteration 1-3 colab and we did from 4 foward alone

### CODE REVIEW

**Comment by sina-kaabi** [comment]
iteration 1-3 done. worked on 4 onwards alone will update soon with status of code in seperate fork.

File: routes/celebrities.routes.js [hunk 1]
```diff
@@ -26,7 +26,16 @@ router.get('/',(req,res) => {
...
+router.post('/:id/delete', (req, res, next) => {
+    const {id} = req.params
 
+    Celebrity
+      .findByIdAndDelete(id)
+      .then (deletedCelebrity => {
+        res.redirect(`/celebrities`)
+      })
+      .catch(err => console.log(err))
+  });
```
Comment (TA-Remote) [inline routes/celebrities.routes.js:38]: inside the promise, if you are not using the variable - you do not need to write it:

```
      .then (()=> {
        res.redirect(`/celebrities`)
      })
```

File: views/movies/movie-details.hbs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+<h1>{{name}}</h1>
+
+<p>Acts as a: {{occupation}}</p>
+
+<p>Catch Phrase: {{catchPhrase}}</p>
+
+{{/each}}
+
+<a href="/movies">Go back</a>
```
Comment (TA-Remote) [inline views/movies/movie-details.hbs:17]: Make sure you format the code! Much easier to read and looks more organised. Here use indentation for example inside `each`

**Review by TA-Remote — COMMENTED** [review]
Great work on the lab Pedro. Well done, we can see a lot of improvement keep it up!

**Pull request closed without merging**